### DISCUSSION TITLE
excluding certain aks deployments

### DISCUSSION THREAD

**Discussion by thulasidassrinivasan** [vmware-tanzu/velero#3648] [Community support Q&A] [answered]
Is it possible to exclude a certain set of aks deployments alone rather than completely ignoring all deployments when taking backup? If yes how to achieve this ?

**Comment by ashish-amarnath** [comment, marked as answer]
This link https://velero.io/docs/v1.5/backup-reference/#exclude-specific-items-from-backup should have what you are looking for.

**Reply by thulasidassrinivasan** [reply]
Thanks @ashish-amarnath for responding. Is there anyway to exclude deployment using deployments name without adding these labels to them? Say I want to avoid all deployments that has "batch" in them.

**Reply by ashish-amarnath** [reply]
Unfortunately, we don't have regex filtering of resources that get included or excluded from the backup.
